onboard/nav/obstacle_avoidance/simpleAvoidance.cpp

```diff
@@ -102,3 +102,9 @@ Odometry SimpleAvoidance::createAvoidancePoint( Rover* phoebe, const double dist
 
 
 } // createAvoidancePoint()
+
+// Returns true if an obstacle is detected, false otherwise.
+bool SimpleAvoidance::isObstacleDetected(Rover * phoebe) const
+{
+    return phoebe->roverStatus().obstacle().distance >= 0;
+} // isObstacleDetected()
```

Comment:
This is duplicated code which is exactly what this function was trying to avoid. We should only have this function once.